### ISSUE TITLE
Idea: httpurr -list [category #]

### ISSUE THREAD

**Issue by christophberger** [rednafi/httpurr#8]
Idea: optionally filter the list by category (1xx, 2xx, 3xx,...)

```
httpurr -list 4
```

would then list all 4xx codes.

**Comment by rednafi** [comment]
Thanks for the idea. I like this one. However, I'm leaning towards `httpurr -list -cat 4` since it's explicit and communicates the intent better.

**Comment by christophberger** [comment]
It would be more typing and a flag that depends on another one. 

But the intent would indeed be clearer that way.

Or maybe `httpurr -list cat4`?

**Comment by rednafi** [comment]
Ended up going with this:

```sh
httpurr -list -cat 4
```

It returns:

```txt

ᗢ httpurr
==========

Status Codes
------------

------------------ 4xx ------------------

400    Bad Request
401    Unauthorized
402    Payment Required
403    Forbidden
...
```

True, it's a bit extra typing but still not too bad. The [tests] cover the behavior of the dependent flags; so it shouldn't be much of an issue. Released with [v0.1.1]. Thanks much.

[tests]: https://github.com/rednafi/httpurr/blob/93ada773b6756e6009ebe12f8333e26ff5f24aed/src/cli_test.go#L330-L348
[v0.1.1]: https://github.com/rednafi/httpurr/releases/tag/v0.1.1
